doc/rados/operations/pools.rst

```diff
@@ -178,14 +178,14 @@ following:
 Associating a Pool to an Application
 ====================================
 
-Pools need to be associated with an application before use. Pools that will be
-used with CephFS or pools that are automatically created by RGW are
-automatically associated. Pools that are intended for use with RBD should be
-initialized using the ``rbd`` tool (see `Block Device Commands`_ for more
-information).
+Pools need to be associated with an application before they can be used. Pools
+that are intended for use with CephFS and pools that are automatically created
+by RGW are associated automatically. Pools that are intended for use with RBD
+should be initialized with the ``rbd`` tool (see `Block Device Commands`_ for
+more information).
```

Comment:
created automatically?  Remember the "Boldly go" rule ;)

Reply:
This isn't a split infinitive, but I share your zeal for keeping the parts of a complex verb together. So: accepted.